helm/dagster/schema/schema_tests/test_user_deployments.py

```diff
@@ -553,6 +558,41 @@ def test_user_deployment_image(template: HelmTemplate):
     assert image_name == deployment.image.repository
     assert image_tag == deployment.image.tag
 
+    _assert_has_container_context(user_deployments[0])
+
+
+def test_user_deployment_default_config(template: HelmTemplate):
+    deployment = create_simple_user_deployment("foo", include_config_in_launched_runs=None)
+    helm_values = DagsterHelmValues.construct(
+        dagsterUserDeployments=UserDeployments.construct(
+            enabled=True,
+            enableSubchart=True,
+            deployments=[deployment],
```

Comment:
Should have a test case with multiple deployments, since `values.yaml` only applies to the first deployment